<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddFriend</class>
 <widget class="QDialog" name="AddFriend">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>227</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="horizontalFrame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>541</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(38,40,45,255);
border-radius:25%;
border:3px solid white;</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="topMargin">
     <number>5</number>
    </property>
    <item alignment="Qt::AlignRight">
     <widget class="QFrame" name="CloseMinimizeButtons">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="minimizeWindow_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius:10px;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../res-rec.qrc">
           <normaloff>:/icons/icon/remove_24dp_EFEFEF_FILL0_wght400_GRAD0_opsz24.svg</normaloff>:/icons/icon/remove_24dp_EFEFEF_FILL0_wght400_GRAD0_opsz24.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="closeWindowButton_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius:10px;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../res-rec.qrc">
           <normaloff>:/icons/icon/close_24dp_EFEFEF_FILL0_wght400_GRAD0_opsz24.svg</normaloff>:/icons/icon/close_24dp_EFEFEF_FILL0_wght400_GRAD0_opsz24.svg</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QFrame" name="AddFriendWrapper">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>200</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(38,40,45,255);
border:none;</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">color:white;
font-size:20px;</string>
         </property>
         <property name="text">
          <string>Введите никнейм друга</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QFrame" name="FriendsNickPlaceholer">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:rgba(38,40,45,255);</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="FriendsNick_input">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit {
 width:250px;
height:30px;
border: 2px solid #323338;
border-radius: 10px;
background-color:#323338;
font-size:16px;
color:#808994;
text-align:left;
padding-left:15px;
}

</string>
            </property>
            <property name="placeholderText">
             <string>Ник друга...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="AddFriend_button">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>35</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">border:3px solid white;
border-radius:10%;
color:white;
font-size:16px;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../res-rec.qrc">
              <normaloff>:/icons/icon/person_add_24dp_FFFFFF_FILL0_wght400_GRAD0_opsz24.svg</normaloff>:/icons/icon/person_add_24dp_FFFFFF_FILL0_wght400_GRAD0_opsz24.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="Status">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../res-rec.qrc"/>
 </resources>
 <connections/>
</ui>
